src/main/java/org/traccar/web/shared/model/UserSettings.java

```diff
@@ -86,6 +86,7 @@ public double toKnots(double speed) {
         BING_ROAD("Bing Road") { @Override public boolean isBing() { return true; } },
         BING_HYBRID("Bing Hybrid") { @Override public boolean isBing() { return true; } },
         BING_AERIAL("Bing Aerial") { @Override public boolean isBing() { return true; } },
+        BING_OS("Bing OSGB") { @Override public boolean isBing() { return true; } },
```

Comment:
Let's name enum element in a more descriptive way, like `BING_ORDNANCE_SURVEY`, or at least `BING_OSGB`.